go.mod

```diff
@@ -8,7 +8,7 @@ toolchain go1.23.4
 
 require (
 	github.com/benbjohnson/clock v1.3.0 // indirect
-	github.com/btcsuite/btcd v0.24.3-0.20240921052913-67b8efd3ba53
```

Comment:
In lnd as well, rather than using a fixed version, it references a commit in the same way.

Reply:
Ah, this is just a note for my own record. Please disregard.